packages/react-sdk/rollup.config.js

```diff
@@ -0,0 +1,51 @@
+import { defineConfig } from "rollup";
+import typescript from "@rollup/plugin-typescript";
+import { dts } from "rollup-plugin-dts";
+import tsConfigPaths from "rollup-plugin-tsconfig-paths";
+import terser from "@rollup/plugin-terser";
+import filesize from "rollup-plugin-filesize";
+
+export default defineConfig([
+  {
+    input: "src/index.ts",
+    output: {
+      file: "lib/index.js",
+      format: "es",
+      sourcemap: true,
+    },
+    plugins: [
+      tsConfigPaths(),
+      typescript({
+        declaration: false,
+        declarationMap: false,
+      }),
+      terser(),
+      filesize({
+        showMinifiedSize: false,
+      }),
+    ],
+    external: [
+      "@xmtp/content-type-reaction",
+      "@xmtp/content-type-read-receipt",
```

Comment:
intentional?

Reply:
since we're still exporting the config for this content type, it's in this list.